helm/solr-operator/values.yaml

```diff
@@ -0,0 +1,26 @@
+# Default values for solr-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: bloomberg/solr-operator
+  tag: v0.2.1
```

Comment:
It would be amazing if we could auto-update this on every build, like we do with the kustomize stuff here:
https://github.com/bloomberg/solr-operator/blob/master/Makefile#L57

I've tried to automate away any explicit references to a version. So that we can release without having to update versions.

Reply:
Honestly maybe we should just set this to `latest`?

Reply:
I'm good with `latest`  if you don't think it has the capacity to cause any problems

Reply:
I agree that it should be the highest version, and I would love to have some automated scripts to go and update the versions throughout the project everytime a release is made. Until then I'd say we should probably use latest.

It's at least a small improvement we can make in the future. Getting this merged is more important 😃